regression/mint/react-icu/l10nmonster.mjs

```diff
@@ -0,0 +1,39 @@
+export default class ReactConfig {
+    sourceLang = 'en';
+    minimumQuality = 50;
+    qualifiedPenalty = 1;
+    unqualifiedPenalty = 9;
+
+    constructor({ ctx, stores, adapters, filters, normalizers, translators }) {
+        this.source = new adapters.FsSource({
+            globs: [ '**/en/*.json' ],
+            targetLangs: [ 'de', 'ru' ],
+        });
+        this.resourceFilter = new filters.JsonFilter();
+        this.decoders = [ normalizers.iosPHDecoder, normalizers.javaEscapesDecoder ];
+        // this.translationProvider = new translators.TranslationOS({
+        //     baseURL: 'https://api-sandbox.translated.com/v2',
+        //     apiKey: ctx.env.translated_api_key_sandbox,
+        //     serviceType: 'premium',
+        //     quality: 90,
+        //     trafficStore: new stores.FSTrafficStore(),
+        // });
+        // this.translationProvider = new translators.ModernMT({
+        //     apiKey: ctx.env.mmt_api_key,
+        //     quality: 40,
+        //     maxCharLength: 100,
+        // });
+        this.translationProvider = new translators.DeepL({
```

Comment:
Normally we shouldn't have dependencies to 3rd parties at regression. Although that brings the question of how we test integrations...